git-refresher/moving-between-branches.qmd

```diff
@@ -1,18 +1,61 @@
---- 
+---
 title: "Moving between branches"
 ---
 
-...
+This section will guide you through switching branches in your repo.
+
+Switching branches is useful for when we might need to pause our work on one branch and continue working on an issue in another branch. Before you switch branches it is important to commit and push the changes in the branch you're currently in. It's also important to pull changes after you switch to your new branch.
 
 ::: panel-tabset
 ## Git Bash
 
+First you need to update your local repo with the changes made to the remote one. This ensures that you have a full list of remote branches and the latest changes in the code to avoid conflicts. Enter the command below:
+
+`git pull`
+
+In Git Bash, you can switch branches using either of the following options:
+
+`git checkout`
```

Comment:
Let's give an explicit demo here, we want everything to be practically lead. So `git checkout main` and then a line saying "check the git-academy-log.md" file and you'll see your edits have disappeared", then run  `git checkout <github_username>/main` and you'll see your  edits will return.